### PR TITLE
[WIP] Update QtWebKit to 5.212

### PR DESCRIPTION
**Pull request by dirkhh**
All of the work was done by @aquiles2k - all I did was fix the install prefix
and create the pull request so that people can start discussing this proposed
change.

This has only been tested for shared builds and requires posix threads.
x86_64-w64-mingw32.shared.posix.dw2 and i686-w64-mingw32.shared.posix.dw2
appear to work.

### CODE REVIEW

File: src/qtwebkit-1-fixes.patch [hunk 1]
```diff
@@ -2,94 +2,221 @@ This file is part of MXE. See LICENSE.md for licensing information.
...
+@@ -179,9 +179,9 @@
+ sub CygwinPathIfNeeded
+ {
+     my $path = shift;
+-    if ($path && $Config{osname} eq "cygwin") {
++    if ($path && ($Config{osname} eq "cygwin" || $Config{osname} eq "msys")) {
+         if (not $cygwinPathAdded) {
+-            $ENV{PATH} = "$ENV{PATH}:/cygdrive/c/cygwin/bin";
++            $ENV{PATH} = "$ENV{PATH}:/usr/bin";
+             $cygwinPathAdded = 1;
```
Comment (sfuchs79) [inline src/qtwebkit-1-fixes.patch]: For me the patch didn't apply due to a white space issue in this line. In fact the ORIGINAL file has a trailing white space but the patch file doesn't.

**Comment by sfuchs79** [comment]
Beside my single comment above I managed to build MXE with Qt 5.11.2 incl. WebKit and then also managed to build Subsurface. Great!

**Comment by tehnick** [comment]
Some news: https://github.com/mxe/mxe/issues/2258#issuecomment-450441252

**Comment by dirkhh** [comment]
#2268 is the way to go

**Pull request closed without merging**